core/src/main/java/hudson/PluginManagerStaplerOverride.java

```diff
@@ -0,0 +1,29 @@
+package hudson;
+
+import jenkins.model.Jenkins;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Extension point for selectively overriding parts of the {@link PluginManager} views
+ * Anything extending this and registered with an @Extension can replace existing views and define new views.
+ *
+ * It is also possible to add/modify API calls coming via Stapler, but this requires caution.
+ *
+ * In both cases, this is simply done by defining a resource or method that matches the existing one
+ *
+ * @author Sam Van Oort
+ * @since 1.626
+ */
+public abstract class PluginManagerStaplerOverride implements ExtensionPoint {
+
+    public static @CheckForNull PluginManager getManager() {
```

Comment:
Agree in general, but seems a tad silly for a simple trivial convenience getter.  Will add it if there's  a second vote for this.